Makefile

```diff
@@ -253,6 +253,9 @@ all: addon server
 .PHONY: clean
 clean:
 	rm -rf build/ addon/webextension/build/ addon/webextension/manifest.json addon/webextension/_locales/
+
+.PHONY: distclean
+distclean: clean
```

Comment:
Distclean generally gets rid of the support files/dependencies as well as the build stuff. The simple clean is just for the build (or that's how we did it in Loop)